locales/ru_RU/LC_MESSAGES/duckduckgo.po

```diff
@@ -104,7 +104,7 @@ msgstr "%1$sУзнайте больше%2$s."
 
 msgctxt "precise_user_location"
 msgid "%sLearn how we keep your location private%s."
-msgstr "%1$sУзнайте, как мы сохраняем конфиденциальность вашего местоположения%2$s."
+msgstr "%1$sКак мы сохраняем конфиденциальность вашего местоположения%2$s."
```

Comment:
Removed characters to make a sentence one line
![image](https://user-images.githubusercontent.com/63919734/179726229-4e30959e-52c7-423b-8578-e16c4c62eded.png)